<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginBarGuiComp</class>
 <widget class="QWidget" name="CLoginBarGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>237</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserEdit"/>
      </item>
      <item>
       <widget class="QFrame" name="PasswordFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PasswordEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LoginButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="icon">
         <iconset resource="../../../Acf/Include/iqtgui/iqtgui.qrc">
          <normaloff>:/Icons/Ok</normaloff>:/Icons/Ok</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LogoutButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
        <property name="icon">
         <iconset resource="../../../Acf/Include/iqtgui/iqtgui.qrc">
          <normaloff>:/Icons/Shutdown</normaloff>:/Icons/Shutdown</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Acf/Include/iqtgui/iqtgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PasswordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>LoginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UserEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>LoginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
